forem/forem
# fix error that showed unapproved posts in sidebar

## What type of PR is this? (check all applicable)


## Description
This fixes the error that was showing unapproved posts in the sidebar
## Related Tickets & Documents
Closes #20772 

## QA Instructions, Screenshots, Recordings

Go to the admin and edit a Tag to change it to require approval
![image](https://github.com/forem/forem/assets/33267902/04f44bf2-2ee9-46ac-b983-9c9e322341c1)

Now make this tag appear in the front page, to make this go to the Customization>Config menu and add the Tag you want to display here
![image](https://github.com/forem/forem/assets/33267902/a5e154f7-b72c-481e-bda6-b57cc422df80)

Now if you go to the home page it should display only approved posts in this tag. If there is no post its probably because you don't have any approved posts. To approve a post go to the admin panel and go to the menu Content Manager>Posts

Go to the Posts that has the tag you selected and mark it as approved and save

![image](https://github.com/forem/forem/assets/33267902/52112139-5d0c-4e83-b3c6-bc9dd40f6483)

After that this post should appear in the home page

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_
